one-offs/migrate-boms/migrate-boms.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"log"
+	"regexp"
+	"strings"
+
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/option"
+)
+
+var (
+	srcBucket   = flag.String("srcBucket", "halconfig", "The GCS bucket name to read from. Must contain a bom/ directory.")
+	destBucket  = flag.String("destBucket", "halconfig2", "The GCS bucket name to write to.")
+	jsonKeyPath = flag.String("jsonKey", "", "Filepath to JSON key with permission to read --src-bucket and write to the --dest-bucket")
+
+	releaseRegexp = regexp.MustCompile(`1\.[0-9]{1,2}\.[0-9]{1,2}\.yml`)
+)
+
+func main() {
+	flag.Parse()
+
+	ctx := context.Background()
+	storageSvc, err := storage.NewClient(ctx, option.WithCredentialsFile(*jsonKeyPath), option.WithScopes(storage.ScopeFullControl))
+	if err != nil {
+		log.Fatalf("Error generating new storage client: %v", err)
+	}
+
+	iter := storageSvc.Bucket(*srcBucket).Objects(ctx, &storage.Query{Prefix: "bom/"})
+	for obj, err := iter.Next(); err == nil; obj, err = iter.Next() {
+		if releaseRegexp.Match([]byte(obj.Name)) {
```

Comment:
(minor) Could do `MatchString` to avoid cast to `[]byte`

Reply:
Nice catch.